sdk/go.mod

```diff
@@ -7,13 +7,15 @@ require (
 	github.com/ligato/cn-infra v2.2.0+incompatible // indirect
 	github.com/ligato/vpp-agent v2.3.0+incompatible
 	github.com/mesos/mesos-go v0.0.9
+	github.com/networkservicemesh/networkservicemesh/controlplane v0.2.0
 	github.com/networkservicemesh/networkservicemesh/controlplane/api v0.2.0
+	github.com/networkservicemesh/networkservicemesh/k8s v0.0.0-20191010104324-d5bfadd36350
+	github.com/networkservicemesh/networkservicemesh/k8s/api v0.2.0
```

Comment:
Again... we don't want k8s stuff landing in sdk as deps.  See above for the easy fix to the issue :)